aemedge/styles/styles.css

```diff
@@ -674,6 +690,22 @@ a.black {
   color: black;
 }
 
+/* stylelint-disable-next-line selector-class-pattern */
+.section.package-cards-container #package-cards-app .hlmNfh {
+  display: flex;
+  flex-direction: column;
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
+.sc-gZMcBi.qgmph{
+  display: flex;
+}
+
+/* stylelint-disable-next-line selector-class-pattern */
+.sc-ifAKCX.cWMbCw{
+  display: flex;
+}
+
```

Comment:
These styles seem to be redundant?
![image](https://github.com/user-attachments/assets/49b4503b-a313-4e4d-8146-c2cc4953abfc)

Reply:
![image](https://github.com/user-attachments/assets/34c1f20d-7176-40da-9466-184fd0afb3ae)